### PR TITLE
Document access to WebGL constants when using standalone bundled library

### PR DESCRIPTION
**Pull request by zakjan**
Docs for https://github.com/visgl/luma.gl/pull/1469

### CODE REVIEW

File: docs/get-started/using-standalone.md [hunk 1]
```diff
@@ -48,12 +48,14 @@ To use deck.gl in a scripting environment, include the standalone version in a `
...
 <script src="https://unpkg.com/deck.gl@latest/dist.min.js"></script>
+<!-- optional if WebGL constants are needed -->
+<script src="https://unpkg.com/@luma.gl/constants@latest/dist/dist.min.js"></script>
 <!-- optional if mapbox base map is needed -->
 <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.js'></script>
 <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.css' rel='stylesheet' />
 ```
 
-It exposes two global objects `deck` and `luma`. Any exports from the deck.gl core can be accessed by `deck.<Class>`.
+It exposes two global objects `deck` and `luma`. Any exports from the deck.gl core can be accessed by `deck.<Class>`. WebGL constants can be accessed by `luma.GL.<constant>`.
```
Comment (ibgreen) [inline docs/get-started/using-standalone.md:58]: - Looks a bit incomplete, Maybe break into bullets and mention a bit more about what luma constants can be accessed from the luma entry point. 
- While obvious, they can also be accessed from the WebGL context when that is available. Worth pointing out as users might get confused.

**Review by ibgreen — APPROVED** [review]

**Review by Pessimistress — COMMENTED** [review]
There are many ways to work with WebGL constants, including:

- Hard coding a number
- Access from the WebGL Context
- Use any of the [JS libraries](https://www.npmjs.com/search?q=webgl%20constants) that wrap these constants, not necessarily `@luma.gl/constants`.

On the other hand, a number of luma.gl features are not bundled with deck.gl but can be useful to include on demand, e.g. how [this example](https://deck.gl/docs/api-reference/core/post-process-effect) uses `@luma.gl/shadertools`.

The same approach also applies to loaders.gl, like how [this example](https://deck.gl/docs/api-reference/core/orbit-view) uses `@loaders.gl/ply`.

I think the best approach to documenting this is to explain:

- What global objects are exposed by the main deck bundle (`deck`, `luma` and `loaders`)
- That submodules from luma.gl and loaders.gl can be included via script tags
- Link to respective doc pages of luma and loaders

**Comment by ibgreen** [comment]
Sounds good. For what it is worth, WebGPU works with a different system (it does not use OpenGL constants), so I am experimenting with a different setup for luma v9 that would work across both APIs, probably more closely aligned with WebGPU, mapping to constants under the hood on WebGL.

**Comment by chrisgervang** [comment]
Is no longer valid in v9? The [upgrade guide](https://deck.gl/docs/upgrade-guide#lumagl-v9-updates) is recommending to use string constants? https://luma.gl/docs/api-reference/core/parameters

**Comment by donmccurdy** [comment]
In most cases the string constants are all you need with lumagl v9+, so documentation on GL-specific constants should not be necessary.

**Pull request closed without merging**